models/src/main/java/io/sphere/sdk/reviews/queries/ReviewsEndpoint.java

```diff
@@ -0,0 +1,8 @@
+package io.sphere.sdk.reviews.queries;
```

Comment:
@schleichardt and @njacintoct This class looks like a 100% copy of the class in package `commands` - is this intended?